website/project/views/contributor.py

```diff
@@ -482,23 +482,7 @@ def send_claim_email(
     notification_type=NotificationType.Type.NODE_CONTRIBUTOR_ADDED_DEFAULT
 ):
     """
-<<<<<<< HEAD
     Send a claim email to an unregistered contributor or the referrer, depending on the scenario.
-=======
-    Unregistered user claiming a user account as a contributor to a project. Send an email for claiming the account.
-    Either sends to the given email or the referrer's email, depending on the email address provided.
-
-    :param str email: The address given in the claim user form
-    :param User unclaimed_user: The User record to claim.
-    :param Node node: The node where the user claimed their account.
-    :param bool notify: If True and an email is sent to the referrer, an email
-        will also be sent to the invited user about their pending verification.
-    :param int throttle: Time period (in seconds) after the referrer is
-        emailed during which the referrer will not be emailed again.
-    :param str email_template: the email template to use
-    :return
-    :raise http_status.HTTP_400_BAD_REQUEST
->>>>>>> upstream/hotfix/25.18.1
```

Comment:
Nitpicking: can we update the longer/better docstring to match the `send_claim_email` in NR without wiping it out?